ansible/inventory

```diff
@@ -1,7 +1,14 @@
+[k8s:children]
```

Comment:
I _believe_ that I am experiencing a bug since this change.

bootstrapping a cluster from scratch has started my cluster as a single-master using only the first master on the inventory list.

I am investigating on why this is happening.

Reply:
It has to do with the jinja interpretation of this line: https://github.com/raspbernetes/k8s-cluster-installation/blob/80c7e1268cdbec7ee65b786c821bdd863c63e4c8/ansible/roles/cluster/templates/kubeadm-join.yml.j2#L14

Gonna work on a fix.

Reply:
I believe that https://github.com/raspbernetes/k8s-cluster-installation/pull/76/commits/478c2ea499730caa35bf30aa460c1197982a6ee6 fixes this issue